src/components/Login/AuthTextFieldsData.ts

```diff
@@ -0,0 +1,52 @@
+export const LoginTextFields = [
+  {
+    id: 'outline-email',
+    label: 'Email Address',
+    variant: 'outlined',
+    type: 'email',
+  },
+  {
+    id: 'outline-pwd',
+    label: 'Password',
+    variant: 'outlined',
+    type: 'password',
+  },
+];
+
+export const SignUpTextFields = [
+  {
+  id: "firstName",
+  label: "First Name",
+  type: "text",
+  placeholder: "Enter your first name",
+  required: true,
+  },
```

Comment:
🤔 @AlvisonHunterArnuero, we haven't set up a linter for this project, have we?

Reply:
not sure...